### ISSUE TITLE
Missing Slack channel integration 

### ISSUE THREAD

**Issue by mochadwi** [TeamCodeStream/codestream#767]
<img width="838" alt="Screen Shot 2021-11-17 at 9 24 29 AM" src="https://user-images.githubusercontent.com/7057959/142101172-aea72bca-a402-43f7-82a6-cf4bbef479b9.png">

Previously the slack channel selection exist.

We've tried re-integrate our Slack workspace, but still not action to choose channel

```
Android Studio: Arctic Fox
AGP: 7.2
CodeStream: 12.1.0+206
Kotlin: 1.5.21
Kotlin Plugin: 1.6
```

**Comment by TarunavBA** [comment]
Hey dude click on that drop-down near select a channel. And try if it works.

~ 🤖

**Comment by bcanzanella** [comment]
hey @mochadwi sorry you're hitting a snag, could you send your logs to support@codestream.com. Instructions for finding your logs are here https://docs.codestream.com/userguide/faq/client-logs/

**Comment by planteater** [comment]
@mochadwi - I might also suggest disconnecting from Slack, restarting your IDE, and then connecting again.

**Comment by mochadwi** [comment]
Apologize for the late response. 

Seems working now. 🎉


Looks like due to my internet connection issue making it fails to load and post to slack channel

**Comment by mochadwi** [comment]
Weirdly, this happened again.

**Comment by mochadwi** [comment]
will try to collect log later, will inform you again

**Comment by mochadwi** [comment]
```
2021-11-21 19:04:25,950 [93531028]   WARN - agent.CodeStreamLanguageClient - [2021-11-21 12:04:25:896] [d19] SlackProvider.getChannels — failed • 5331 ms

.... // mostly our slack information, such as username, etc

org.eclipse.lsp4j.jsonrpc.ResponseErrorException: Request codestream/provider/channels failed with message: Cannot create property 'response_metadata' on string
```

Does this internet connection? I've also emailed the full logs

**Comment by bcanzanella** [comment]
hi @mochadwi we've added additional logging in the latest version of CodeStream for JetBrains (version 12.3.1). Could you try again with that version? If there are any issues, could you once again send your JetBrains logs to support@codestream.com? Thanks!

**Comment by planteater** [comment]
@mochadwi - This should be fixed in v12.7
